product_pricelist_alternative/__manifest__.py

```diff
@@ -3,14 +3,15 @@
 
 {
     "name": "Product Pricelist Alternative",
-    "version": "18.0.1.0.0",
+    "version": "18.0.1.1.0",
```

Comment:
@vvrossem If you don't provide migration scripts, this is not needed as updated by the bot.